Terasology/Health
# For solving the multi block non healing bug

https://github.com/Terasology/Health/pull/12 rebased on master.
Added some loggers for finding whats going wrong

How to test:
1. Start a game with BSG and Health module.
2. Hit many blocks consecutively.
3. See that all of them regenerate, and not any gets "stuck" in damaged state.
